phenopacketlab-core/src/main/java/org/monarchinitiative/phenopacketlab/core/ConceptConstantsServiceImpl.java

```diff
@@ -27,10 +27,10 @@ class ConceptConstantsServiceImpl implements ConceptConstantsService {
                                 List<IdentifiedConcept> allelicStateConstants,
                                 List<IdentifiedConcept> lateralityConstants,
                                 List<IdentifiedConcept> modifierConstants,
-                                Optional<SubtreeNode> modifierTreeConstants,
+                                SubtreeNode modifierTreeConstants,
```

Comment:
In the new endpoints I added to the backend (merged in dev now), tnm finding and disease stage, they also return SubTreeNodes, I think I might have put the Optional in there too. We need to make sure to refactor those the same way as the above